distribution/docs/src/main/resources/_contents/_installing/installing-intro-contents.adoc

```diff
@@ -314,6 +314,9 @@ During ${branding} installation, the major directories and files shown in the ta
 |`data/log/security.log`
 |Log file that records user interactions with the system for auditing purposes.
 
+|`data/log/solr.log`
+|Log file for any info coming from solr.
```

Comment:
I would reword to "from Standalone Solr Server".  I think that is how we usually refer to it in documentation.  Does that sound right @ricklarsen?